src/merkle/smt/full/tests.rs

```diff
@@ -59,8 +59,12 @@ fn test_smt_insert_at_same_key_2() {
     // The most significant u64 used for both keys (to ensure they map to the same leaf)
     let key_msb: u64 = 42;
 
-    let key_already_present: RpoDigest =
-        RpoDigest::from([2_u64.into(), 2_u64.into(), 2_u64.into(), Felt::new(key_msb)]);
+    let key_already_present: RpoDigest = RpoDigest::from([
+        2_u64.try_into().unwrap(),
+        2_u64.try_into().unwrap(),
+        2_u64.try_into().unwrap(),
```

Comment:
Since the values are small, we can probably do `2_u32.into()`.

This also applies to most other changes in this file.